test/e2e_node/e2e_service.go

```diff
@@ -190,7 +190,9 @@ func (es *e2eService) startEtcd() (*killCmd, error) {
 		}
 		etcdPath = defaultEtcdPath
 	}
-	cmd := exec.Command(etcdPath)
```

Comment:
The plan for `etcd3` is a bit different. It's not being shipped in the OS but rather as a container. For this code, it would probably be easiest to handle that difference in the CoreOS image config rather than the go code here. If we'd like to have the tests running against etcd3 on CoreOS I'd be happy to handle that in a followup PR.